app/src/main/java/framgia/vn/framgiacrb/ui/activity/SettingActivity.java

```diff
@@ -57,15 +63,20 @@ protected void onPostCreate(Bundle savedInstanceState) {
         mToolbar.setNavigationOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                setResult(Constant.RequestCode.SETTING);
+                Intent data = new Intent();
+                data.putCharSequenceArrayListExtra(Constant.Intent.INTENT_LIST_CALENDAR,
+                    mListCalendar);
+                setResult(Constant.RequestCode.SETTING, data);
```

Comment:
Dạ vì lúc e start activity for result e truyền vào 1 request code, nên lúc e setResult e cũng truyền vào 1 requestcode như thế ạ